cartridge/webui/api-auth.lua

```diff
@@ -76,7 +76,7 @@ local function get_auth_params()
 
     local username = auth.get_session_username()
     local user = username and auth.get_user(username)
-    if user ~= nil and user.fullname ~= nil then
+    if user and (user.fullname or '') ~= '' then
```

Comment:
This check is quite strange.
  - Check will be broken if fullname is box.NULL
  - And seems it's easier to check that string length has non-zero length: `user ~= nil and user.fullname ~= nil and #user.fullname > 0`

Reply:
Thanks, fixed!